src/Toolkit/Toolkit/GeoViewController.cs

```diff
@@ -0,0 +1,279 @@
+
+        /// <summary>
+        /// Raised when the attached <see cref="GeoView"/> loads into the active view.
+        /// </summary>
+        /// <param name="geoView">GeoView that was loaded</param>
+        protected virtual void OnGeoViewLoaded(GeoView geoView)
+        {
+        }
+
+        /// <summary>
+        /// Raised when the attached <see cref="GeoView"/> unloads from the active view.
+        /// </summary>
+        /// <param name="geoView">GeoView that was unloaded</param>
+        protected virtual void OnGeoViewUnloaded(GeoView geoView)
+        {
+        }
+
+        private void GeoView_Loaded(GeoView geoView)
+        {
+            Debug.Assert(geoView == ConnectedView && ConnectedView is not null);
+            OnGeoViewLoaded(geoView);
+        }
+
+        private void GeoView_Unloaded(GeoView sender)
+        {
+            Debug.Assert(sender == ConnectedView && ConnectedView is not null);
+            OnGeoViewUnloaded(sender);
+        }
+
+        #region Viewpoints
+
+        /// <inheritdoc cref="GeoView.GetCurrentViewpoint(ViewpointType)"/>
+        public virtual Viewpoint? GetCurrentViewpoint(ViewpointType viewpointType) => ConnectedView?.GetCurrentViewpoint(viewpointType);
+
+        /// <inheritdoc cref="GeoView.SetViewpointAsync(Viewpoint)"/>
+        public virtual Task<bool> SetViewpointAsync(Viewpoint viewpoint) => ConnectedView?.SetViewpointAsync(viewpoint) ?? Task.FromResult(false);
+
+        /// <inheritdoc cref="GeoView.SetViewpointAsync(Viewpoint, TimeSpan)"/>
+        public virtual Task<bool> SetViewpointAsync(Viewpoint viewpoint, TimeSpan duration) => ConnectedView?.SetViewpointAsync(viewpoint, duration) ?? Task.FromResult(false);
+
+        /// <inheritdoc cref="GeoView.SetViewpoint(Viewpoint)"/>
+        public virtual void SetViewpoint(Viewpoint viewpoint) => ConnectedView?.SetViewpoint(viewpoint);
+
+        #endregion Viewpoints
+
+        #region Identify
+        
+        /// <inheritdoc cref="GeoView.IdentifyLayersAsync(Point, double, bool, CancellationToken)"/>
+        public virtual Task<IReadOnlyList<IdentifyLayerResult>> IdentifyLayersAsync(Point screenPoint, double tolerance, bool returnPopupsOnly = false, CancellationToken cancellationToken = default) =>
+            ConnectedView?.IdentifyLayersAsync(screenPoint, tolerance, returnPopupsOnly, cancellationToken) ??
+                Task.FromResult<IReadOnlyList<IdentifyLayerResult>>(new List<IdentifyLayerResult>().AsReadOnly());
+        
+        /// <inheritdoc cref="GeoView.IdentifyLayerAsync(Layer, Point, double, bool, CancellationToken)"/>
+        public virtual Task<IdentifyLayerResult> IdentifyLayerAsync(Layer layer, Point screenPoint, double tolerance, bool returnPopupsOnly = false, CancellationToken cancellationToken = default) =>
+            ConnectedView?.IdentifyLayerAsync(layer, screenPoint, tolerance, returnPopupsOnly, cancellationToken) ??
+                Task.FromResult<IdentifyLayerResult>(null!);
```

Comment:
It's a shame we have to return null for non-nullable task.  But I'm not sure if adding `IdentifyLayerResult.CreateEmpty(ILayerContent)` to the public API is worth it just for MVVM purity.

Maybe this method (and IdentifyGraphicsOverlayAsync) can return a task with a nullable result instead?